pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -138,7 +182,7 @@ type KogitoApp struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   KogitoAppSpec   `json:"spec,omitempty"`
+	Spec   *KogitoAppSpec  `json:"spec"`
```

Comment:
could you explain why this is being made a pointer? i don't see any corresponding code to indicate a reason for the change changes which is why i ask

Reply:
the pointer makes the spec required in the crd. But thinking again, is a little too excessive. I believe that there's no need to do that.